src/constants/chains.ts

```diff
@@ -1,22 +1,20 @@
-/**
- * List of all the networks supported by the Uniswap Interface
- * TODO(INFRA-90): Eventually this may be derived from sdk-core.
- */
+import { SupportedChainId as SdkSupportedChainId } from '@uniswap/sdk-core'
```

Comment:
This should be accompanied by a lint rule that prevents importing SupportedChainId elsewhere to prevent regressions, which would be appropriate to add as an optional rule in our shared uniswap eslint config.